conda/exceptions.py

```diff
@@ -556,7 +556,7 @@ def print_conda_exception(exception):
         stdoutlogger.info(json.dumps(exception.dump_map(), indent=2, sort_keys=True,
                                      cls=EntityEncoder))
     else:
-        stderrlogger.info("\n\n%r", exception)
+        stderrlogger.info("\n%r", exception)
```

Comment:
Yes, there were double new lines before and after the message previously.  Now there's only a single newline surround the message.